CHANGELOG.md

```diff
@@ -51,6 +52,17 @@
  - [0.0.5](#005)
  - [0.0.4 and prior](#004-and-prior)
 
+## [2.7.0]
```

Comment:
We are going to release 2.7.0 today. I think the changelog about what you have done should appear in 2.8.0.